cms/static/sass/partials/_variables.scss

```diff
@@ -278,3 +278,5 @@ $body-line-height: golden-ratio(.875em, 1);
 // carried over from LMS for xmodules
 $action-primary-active-bg: #1AA1DE !default; // $m-blue
 $very-light-text: $white !default;
+
+$color-background-bluish: rgb(242, 248, 251);
```

Comment:
A couple of comments:
1. All variables should have ``!default`` so that they can be overridden by a theme
2. Variables shouldn't include color names (e.g. "bluish") because in some themes they won't be that color. Can you choose a semantic name. Even something as simple as ``$color-background-alternate`` would be an improvement, if there isn't a clear rationale for why we use this blue background.